### ISSUE TITLE
Skip link is missing

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/parabol#5496]
## Issue - Bug

A [skip link](https://webaim.org/techniques/skipnav/) is a visually hidden link that appears as the first focusable element in the website's HTML. It can only be interacted with by manually moving focus to it with a keyboard or similar devices. When it receives focus, the link is visually revealed.

A skip link allows screen reader and keyboard users to navigate directly to the main content of the page, skipping the main navigation bar.

Related WCAG Success criteria: [2.4.1 Bypass Blocks](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=241#bypass-blocks)

Some examples of skip links (try loading the page and pressing <kbd>TAB</kbd> on your keyboard):

[amazon.com](https://www.amazon.com/)
[nytimes.com](https://www.nytimes.com/)
[target.com](https://www.target.com/)
[whitehouse.gov](https://www.whitehouse.gov/)
[gov.uk](https://www.gov.uk/)

### Acceptance Criteria
Screen reader and keyboard users can:
   - Find a skip link as the first tab stop of the web page
   - Interact with it to move focus to the main content area of the web page

**Comment by enriquesanchez** [comment]
Closed by #5783
